postcard_creator/token.py

```diff
@@ -71,6 +67,11 @@ def __init__(self, _protocol='https://'):
                           'Version/4.0 Chrome/52.0.2743.98 Mobile Safari/537.36',
             'Origin': '{}account.post.ch'.format(self.protocol)
         }
+        self.request_headers = {
+            'User-Agent': 'Mozilla/5.0 (Linux; Android 6.0.1; wv) ' +
+                          'AppleWebKit/537.36 (KHTML, like Gecko) ' +
+                          'Version/4.0 Chrome/52.0.2743.98 Mobile Safari/537.36',
+        }
```

Comment:
merge request_headers and headers

Reply:
alternatively rename them to more meaningful names such as swissid_headers and legacy_headers

Reply:
Good point, see https://github.com/wp99cp/postcard_creator_wrapper/commit/3693ace8325f875dc07069d340d766ac1875df23.